src/controllers/loginController.js

```diff
@@ -0,0 +1,30 @@
+const loginService = require('../services/login');
+
+const loginController = function loginController(req, res) {
```

Comment:
you could write it in the newer way like this:
const loginController = (req, res) => {

try it out. if it works change it.